### DISCUSSION TITLE
Getting "Refused to load the script" when try to load comments.

### DISCUSSION THREAD

**Discussion by mdrathik** [timlrx/tailwind-nextjs-starter-blog#381] [Q&A] [answered]
Anyone can help me to get rid of this error. 
https://prnt.sc/26sly1d

This template already using this , but don't know why its giving the error. 
`const ContentSecurityPolicy = 
  default-src 'self';
  script-src 'self' 'unsafe-eval' 'unsafe-inline' giscus.app;
  style-src 'self' 'unsafe-inline' *.googleapis.com cdn.jsdelivr.net;
  img-src * blob: data:;
  media-src 'none';
  connect-src *;
  font-src 'self' fonts.gstatic.com cdn.jsdelivr.net;
  frame-src giscus.app`

**Comment by nelsonlaidev** [comment, marked as answer]
I solved it like this and this works. But I don't know if this is the right way to do it.

```js
const ContentSecurityPolicy = `
  default-src 'self';
  script-src 'self' 'unsafe-eval' 'unsafe-inline' giscus.app scripts.simpleanalyticscdn.com www.googletagmanager.com utteranc.es;
  style-src 'self' 'unsafe-inline' *.googleapis.com cdn.jsdelivr.net;
  img-src * blob: data:;
  media-src 'none';
  connect-src *;
  font-src 'self' fonts.gstatic.com cdn.jsdelivr.net;
  frame-src giscus.app
`;
```

**Reply by AlexanderZeilmann** [reply]
Yes, this is the correct way 👍

**Comment by mdrathik** [comment]
Thanks a lot, both of you Guyz. @TszHong0411  & @AlexanderZeilmann 
Basically, I missed adding utteranc.es.
It's fixed
